content/use-cases/use-an-escrow-as-a-smart-contract.md

```diff
@@ -0,0 +1,149 @@
+
+    To handle a no-show scenario, the participants configure the escrow to be eligible for cancellation after a specific time. Typically, this cancellation time should be after the time at which the party planner is expected to finish the escrow. The the cancellation time in this way helps prevent the party host from maliciously cancelling the escrow before the party planner can finish the escrow.
+
+    The conditionally-held escrow provides the functionality required to handle the terms of this smart contract by putting the ability to finish the escrow solely in the hands of the neutral party planner (oracle).
+
+    - *[Time-held escrow](send-a-time-held-escrow.html)*: A time-held escrow is not the best choice for this scenario because the party band could be a no-show. If the party band is scheduled to finish their set at 11 p.m., the participants could agree to allow the escrow to finish and release the XRP at 11 p.m. At 11 p.m., any participant, including the party band, can finish the escrow and release the XRP, even if the band didn't show up. This does not meet the terms of the smart contract.
+
+    The participants could agree to allow the cancellation of the escrow before 11 p.m. to accommodate the no-show scenario. However, consider that the party band could show up and play their set, but the party host cancels the escrow before a participant can finish the escrow at 11 pm.
+
+- Should the escrow disallow fulfillment until a specific time?
+
+  The participants agree to disallow the escrow from being fulfilled until 11 p.m. the night of the party, which is when the party band is scheduled to complete their set. ***TODO: not necessary, but just an extra precaution? Or just unnecessary?***
+
+- Should the participants be able to cancel the escrow after a specific time period if the condition is not fulfilled?
+
+  The participants agree that the escrow can be cancelled after 12 noon the day after the party. This gives the party planner enough time to finish the escrow, if the party band fulfills their end of the contract. After cancellation, the locked XRP returns to the party host's account.
+
+  This timing also prevents the party host from maliciously cancelling the escrow before the party planner can finish the fulfilled contract.
+
+- How much XRP should the escrow to lock up and potentially pay?
+
+  The participants agree that the escrow should lock up and potentially pay 2000 XRP, which is the party band's fee.
+
+- Which XRP Ledger account should the escrow lock up and potentially pay XRP out of?
+
+  The participants agree that the escrow should lock up and potentially pay XRP out of the party host's XRP Ledger account.
+
+- Which XRP Ledger account should the escrow lock up and potentially pay XRP into?
```

Comment:
I think this sentence reads more clearly if you remove "lock up and" from it:

> Which XRP Ledger account should the escrow potentially pay XRP to?

(Also, I like "pay XRP to" rather than "pay XRP into" here, but that's not that important.)